src/components/reusables/SummaryPage/SummaryPage.tsx

```diff
@@ -34,13 +38,15 @@ export interface SummaryPageProps {
   successfulDelegations?: ApiDelegation[];
   restartProcessPath: string;
   pageHeaderText: string;
+  headerIcon: React.ReactNode;
+  headerColor?: PageColor;
   topListText?: string;
   bottomListText?: string;
   bottomText?: string;
-  mainButton?: React.ReactNode;
-  complementaryButton?: React.ReactNode;
-  headerIcon: React.ReactNode;
-  headerColor?: PageColor;
+  nextButtonClick?: () => void;
+  nextButtonDisabled?: boolean;
+  nextButtonLoading?: boolean;
+  showNavigationButtons?: boolean;
```

Comment:
Technically, setting this to true still gives you a navigation button, it just takes you back to the overview instead.
I'm not entirely sure what to call it instead though, and it's a nitpick that might just be born from my distaste for the SummaryPage.. 😅

Reply:
Yeah, but that's not clear when you just look at the interface of the SummaryPage component by itself.
But like I said, I'm not sure how to fix this other than renaming it to hideBackButton or something, but I'm not sure that's the right way either since the primary button here is a different one depending on this flag. So maybe this is actually a layout-switch or something?

But again, it's a nitpick born from the Frankensteinien premise of the component itself, so I don't think it's worth a lot of effort when I don't even have a good solution to offer.
You can probably just leave it as it is.

Reply:
Yeah I see your point. If there's a lot of issues with working with the component in the future I'll just delete the whole component and separate ConfirmationPage and ReceiptPage more.